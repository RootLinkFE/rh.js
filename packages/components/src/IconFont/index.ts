import { createFromIconfontCN } from '@ant-design/icons';

// !!! 如需要添加图标，请联系 giscafer 邀请加入协助
const IconFont = createFromIconfontCN({
  scriptUrl: '//at.alicdn.com/t/font_2875265_3hj3tfxeec1.js',
});

export default IconFont;
