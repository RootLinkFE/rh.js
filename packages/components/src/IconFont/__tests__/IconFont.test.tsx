import React from 'react';
import TestRenderer from 'react-test-renderer';
import { createRenderer } from 'react-test-renderer/shallow';
import IconFont from '../index';
// 参考：https://github.com/Jacky-Summer/monki-ui
it('IconFont renders correctly', () => {
  const renderer = createRenderer();
  const icon = <IconFont type="rh-icon-node-channel" />;
  renderer.render(icon);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});
