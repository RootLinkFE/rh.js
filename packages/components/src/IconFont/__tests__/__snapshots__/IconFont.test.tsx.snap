// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconFont renders correctly 1`] = `
<AntdIcon>
  <use
    xlinkHref="#rh-icon-node-channel"
  />
</AntdIcon>
`;
